### PR TITLE
feat: a23-stella-studio

### PR DESCRIPTION
**Pull request by valentinavargas31**

### CODE REVIEW

File: src/mm/A23-Stella-Studio/styles.css [hunk 1]
```diff
@@ -0,0 +1,250 @@
...
+  border-radius: 25px;
+}
+
+#btn-primary {
+  background-color: #333;
+  color: #fff;
+}
+
+#btn-secondary {
+  background-color: #d7d7d7;
```
Comment (Iregui44) [inline src/mm/A23-Stella-Studio/styles.css:94]: This botton is no align with the mobile mockup it mos be down and centered.

File: src/mm/A23-Stella-Studio/index.html [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        </span>
+
+        <div class="content">
+          <div class="buttons">
+            <a href="" id="btn-primary">Get Started</a>
+            <a href="" id="btn-secondary">Get Quote</a>
+          </div>
+          <div class="tags">
+            <ul>
+              <li><a href="">/Web Design</a></li>
```
Comment (Iregui44) [inline src/mm/A23-Stella-Studio/index.html:48]: For better practices when there is no link to go you have to addd a "#"

File: src/mm/A23-Stella-Studio/styles.css (hunk 1 above)
Comment (valentinavargas31) [reply src/mm/A23-Stella-Studio/styles.css:94]: corrected the button is wider, and is centered in the final part of the text as in the reference image

File: src/mm/A23-Stella-Studio/index.html (hunk 2 above)
Comment (valentinavargas31) [reply src/mm/A23-Stella-Studio/index.html]: corrected

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**